trello/search.md

```diff
@@ -0,0 +1,82 @@
+# Searching in Trello
+
+## Keyboard Shortcuts
+
+`/` puts the cursor in the search box
+`b` opens the boards menu
+`f` opens the card filter menu to search through cards on a single board
+`q` toggles "cards assigned to me"
+`x` clears all active card filters
```

Comment:
These keyboard shortcuts all run into one line at the moment:

![image](https://user-images.githubusercontent.com/18957/63268870-14ffda80-c28d-11e9-8d46-392d6d008256.png)

There needs to be an empty line between each one for them to be on separate lines.

Reply:
Thanks. Preview in vscode didn't show that issue.

Reply:
Ahh, I think GitHub's interpretation of Markdown differs from the standard - this looks like it might be useful: https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles